t/t3100-flux-in-flux.t

```diff
@@ -64,15 +64,15 @@ test_expect_success "instance-level attribute = 0 in test instance" '
 '
 
 test_expect_success "instance-level attribute = 1 in first subinstance" '
-        flux mini run flux start ${ARGS} \
-            flux getattr instance-level >level1.out &&
+	flux mini run flux start ${ARGS} \
+		flux getattr instance-level >level1.out &&
 	echo 1 >level1.exp &&
 	test_cmp level1.exp level1.out
 '
 
 test_expect_success "instance-level attribute = 2 in second subinstance" '
-        flux mini run flux start \
-	    flux mini run flux start ${ARGS} \
+	flux mini run flux start \
+		flux mini run flux start ${ARGS} \
```

Comment:
I'm fine with these changes, but in general I think indenting a couple spaces in a line continuation (after the initial tab) should be fine in sharness tests and not considered inconsistent use of tabs/spaces going forward. Using a smaller indent at a line continuation can help avoid lines getting too long in many situations. (I do agree, however, that the initial indent should always be a tab in sharness tests.)

Reply:
OK that is reasonable.